batch/batch/cloud/gcp/worker/disk.py

```diff
@@ -1,8 +1,9 @@
 import logging
 from typing import Dict, Optional
 
-from hailtop.utils import check_shell_output, LoggingTimer, retry_all_errors_n_times
+from hailtop.utils import check_shell_output, retry_all_errors_n_times
 from hailtop.aiocloud import aiogoogle
+from hailtop.utils.utils import LoggingTimer
```

Comment:
This import can be undone. To change this file back to what it currently is in `main`, do `git checkout main batch/**/disk.py`